ts/src/hyperliquid.ts

```diff
@@ -763,6 +763,125 @@ export default class hyperliquid extends Exchange {
         return this.filterByArrayTickers (result, 'symbol', symbols);
     }
 
+    async fetchFundingRates (symbols?: Strings, params?: {}): Promise<FundingRates> {
```

Comment:
@cheunhong next time please try to follow the same approach we use in other exchanges, optional are not supported because we have to transpile this code to different languages (py,PHP,c#)

Reply:
oh yea sorry it was a mistake on my side.

Reply:
@cheunhong  np 🙂